appveyor.yml

```diff
@@ -125,6 +125,9 @@ install:
   - copy C:\Python27\python.exe C:\Python27\python2.7.exe
   - set PATH=C:\Python27;%PATH%
 
+  # Run unit tests for bootstrap.py
+  - python src\bootstrap\bootstrap_test.py
```

Comment:
In `.travis.yml` we can run custom commands before the `script` phase with `before_script`, it seems that the equivalent phase in `appveyor.yml` is `before_build`.